pcsx2/gui/FrameForGS.cpp

```diff
@@ -557,6 +557,7 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 #endif
 
 	double fps = wxGetApp().FpsManager.GetFramerate();
+	float per = gsRegionMode == Region_NTSC ? (fps * 100) / EmuConfig.GS.FramerateNTSC.GetWhole() : (fps * 100) / EmuConfig.GS.FrameratePAL.GetWhole(); 
```

Comment:
@avih 

> GetWhole() sounds suspiciously integer-ish. Are you sure it's returning 59.94 for NTSC?

It seemed to only return the integer part, fixed it with ```ToFloat()```

>Also, did you test that if you modify this config value at the INI file then the display reflects it correctly? (e.g. if you set FramerateNTSC=45.5 at PCSX2_vm.ini and you run an NTSC game - then it should show 100% speed when without the patch the fps would display 45.5).

Yes, modifications at INI file does get reflected.  

>Also, I think displaying 1 decimal point is enough. Like, does anyone really cares if it's 100.02% or 100.03%? also I don't think PCSX2 is stable enough anyway to keep the 2nd decimal point stable under any condition.

i actually prefer 2 decimal points, It looks a little more professional and better compared to  just one. a personal preference maybe :p